runtime/laos/src/configs/election_phragmen.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+	currency::UNIT, weights, Balance, Balances, BlockNumber, Council, Runtime, RuntimeEvent,
+	Treasury,
+};
+use frame_support::parameter_types;
+#[cfg(not(feature = "fast-mode"))]
+use parachains_common::DAYS;
+#[cfg(feature = "fast-mode")]
+use parachains_common::HOURS;
+use polkadot_runtime_common::CurrencyToVote;
+
+#[cfg(feature = "fast-mode")]
+pub const TERM_DURATION: BlockNumber = 4 * HOURS;
```

Comment:
in order to test locally we might want even lower values for `fast-mode`, what do you think?

Reply:
+1. I suggest a few minutes, or even better, a few blocks. Same thing for `ELECTION_VOTING_LOCK_DURATION`

Reply:
a bit complicated to propose, vote, reject, etc etc in a couple of blocks